### PR TITLE
Footer fix

### PR DESCRIPTION
**Pull request by tul1**
Solving the following issue:
https://github.com/ekumenlabs/ekumenlabs.github.io/issues/150

### CODE REVIEW

**Comment by tul1** [comment]
image

**Comment by tul1** [comment]
@german-e-mas as we talked offline I've spread the fix to page.erb and I tested it and it's working properly: the footer is always at the bottom page. So, this is ready for your review.

**Comment by german-e-mas** [comment]
I've tested these changes. Be careful with the following:

localhost_4567_projects_google html 1

There shouldn't be a scrollbar there. The content has been given an incorrect min-height.

I tried to take a look at the issue. Take a look at [my branch](https://github.com/ekumenlabs/ekumenlabs.github.io/tree/footer-german). I was reading [this article](https://css-tricks.com/couple-takes-sticky-footer/) and trying to implement one of its solutions. The idea is to have the header+content+footer heights to be at least 100vh. It would be ideal that the footer had a constant height to calc this in a cleaner way.

**Comment by german-e-mas** [comment]
Some other things to consider: `html` element didn't have [height at 100%](https://stackoverflow.com/questions/6654958/make-body-have-100-of-the-browser-height).

**Comment by tul1** [comment]
Thanks @german-e-mas !

File: source/stylesheets/main.scss [hunk 1]
```diff
@@ -50,6 +51,10 @@ h1, h2, h3, h4, h5, h6 {
   text-shadow: 0 1px 1px rgba(0, 0, 0, 0.4);
 }
 
+div.body-container {
+  min-height: calc(100vh - 203px - #{$navbar-height});
```
Comment (german-e-mas) [inline source/stylesheets/main.scss]: You can add a comment above, stating that the 203px are the `max-height` of the footer (please verify just in case).

Comment (tul1) [reply source/stylesheets/main.scss]: Done @german-e-mas . Let me know if you approve the changes to merge this branch.

**Pull request merged**